aspnetcore/blazor/get-started.md

```diff
@@ -16,12 +16,15 @@ By [Daniel Roth](https://github.com/danroth27) and [Luke Latham](https://github.
 
 Get started with Blazor:
 
-1. Install the latest [.NET Core 3.0 SDK](https://dotnet.microsoft.com/download/dotnet-core/3.0) release.
+1. Install tooling:
 
-1. Install the [Blazor WebAssembly](xref:blazor/hosting-models#blazor-webassembly) template by running the following command in a command shell:
```

Comment:
I think we should leave this step as is for the 3.0 doc set, but update this step to install 3.1 Preview for the 3.1 Preview doc set. In the 3.0 doc set we can then handle installing 3.1 Preview as part of the steps to install Blazor WebAssembly.